subsys/bluetooth/controller/include/ll_feat.h

```diff
@@ -72,13 +72,13 @@
 #define LL_FEAT_BIT_MIN_USED_CHAN 0
 #endif /* !CONFIG_BT_CTLR_MIN_USED_CHAN */
 
-#if defined(CONFIG_BT_CTLR_PHY_2M)
+#if defined(CONFIG_BT_PHY_UPDATE) && defined(CONFIG_BT_CTLR_PHY_2M)
```

Comment:
Changes in this file not required if Kconfig ensures that PHY_2M and PHY_CODED never gets defined.

Reply:
I know, I did this first, and though I should do it through Kconfig instead.
I don't see the harm in keeping it here though.